src/doc/rustc/src/platform-support/aarch64-unknown-linux-gnu.md

```diff
@@ -0,0 +1,50 @@
+# `aarch64-unknown-linux-gnu`
+
+**Tier: 1 (with Host Tools)**
+
+Target for 64-bit little endian Armv8-A Linux 4.1+ programs using glibc 2.17+.
+
+## Target maintainers
+
+- [@rust-lang/arm-maintainers][arm_maintainers] ([rust@arm.com][arm_email])
+
+[arm_maintainers]: https://github.com/rust-lang/team/blob/master/teams/arm-maintainers.toml
+[arm_email]: mailto:rust@arm.com
+
+## Requirements
+
+Building the target itself requires a 64-bit little endian ARMv8-A compiler that is supported by
```

Comment:
Which capitalization is correct, "Armv8-A" from line 5, "ARMv8-A" from line 16 here, or both?

(This also happens in `armv7-unknown-linux-gnueabi.md`)